src/updatedb.py

```diff
@@ -181,29 +188,30 @@ def uploadResults(data, branch, revision, date):
         slave = data1['machine_name']
 
         if (len(data1.get("logs"))):
-            log = data1.get("logs", [])[0].get("url", "")
-        elif (data2):
-            log = data2.get("blob", {}).get("logurl", "")
+            logfile = data1.get("logs", [])[0].get("url", "")
 
         # Insert into MySQL Database
-        sql = """insert into testjobs (id, log, slave, result,
+        sql = """insert into testjobs (log, slave, result,
                                        duration, platform, buildtype, testtype,
                                        bugid, branch, revision, date,
                                        failure_classification)
-                             values ('%s', '%s', '%s', '%s', %s,
+                             values ('%s', '%s', '%s', %s,
                                      '%s', '%s', '%s', '%s', '%s',
                                      '%s', '%s', %s)""" % \
-              (_id, log, slave, _result, \
+              (logfile, slave, _result, \
                duration, platform, buildtype, testtype, \
                bugid, branch, revision, date, failure_classification)
 
         try:
             cur.execute(sql)
+            count += 1
         except MySQLdb.IntegrityError:
             # we already have this job
+            logging.warning("sql failed to insert, we probably have this job: %s" % sql)
```

Comment:
Without a primary key, this should never happen. I guess there is no harm in leaving this here in case we change things around later.